internal/provider/schema_gitlab_project_milestone.go

```diff
@@ -0,0 +1,121 @@
+package provider
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/xanzy/go-gitlab"
+)
+
+func gitlabProjectMilestoneGetSchema() map[string]*schema.Schema {
+	validMilestoneStates := []string{"active", "closed"}
+
+	return map[string]*schema.Schema{
+		"iid": {
+			Description: "The ID of the milestone only in one project.",
```

Comment:
```suggestion
			Description: "The ID of the project's milestone.",
```

(same as in the API documentation)

Reply:
Yeap